devel/cmake/Portfile

```diff
@@ -36,18 +36,18 @@ subport cmake-devel {
     PortGroup       obsolete 1.0
     replaced_by     cmake
     version         20200323-8638c49f
-    revision        2
+    revision        0
```

Comment:
undo this ... this section is not to be touched except for removal entirely (which is coming)